pkg/kubelet/volumemanager/cache/desired_state_of_world.go

```diff
@@ -392,62 +392,66 @@ func (dsw *desiredStateOfWorld) AddPodToVolume(
 	return volumeName, nil
 }
 
-func (dsw *desiredStateOfWorld) getSELinuxLabel(volumeSpec *volume.Spec, seLinuxContainerContexts []*v1.SELinuxOptions) (string, bool, error) {
-	var seLinuxFileLabel string
-	var pluginSupportsSELinuxContextMount bool
+func (dsw *desiredStateOfWorld) getSELinuxLabel(volumeSpec *volume.Spec, seLinuxContainerContexts []*v1.SELinuxOptions, podSecurityContext *v1.PodSecurityContext) (string, bool, error) {
```

Comment:
Can we document what this function returns? It was bit hard to read.

Reply:
Added a comment.